### PR TITLE
test(osv): Update an expected result

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json [hunk 1]
```diff
@@ -258,7 +258,7 @@
             },
             {
                 "type": "WEB",
-                "url": "http://jinja.pocoo.org/docs/changelog/"
+                "url": "http://jinja.pocoo.org/docs/changelog"
```
Comment (sschuberth) [inline clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json:261]: As the trailing slash goes back and forth, should we normalize this, e.g. by always trimming it?

Comment (fviernau) [reply clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json:261]: can we merge this one now and I'll look into normalization into a PR on top ?

Comment (sschuberth) [reply clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json:261]: Yes, I'll approve. Though the final newline was also again stripped. And the `GoDep` test still fails. But let's try to solve one problem at a time here.

**Review by sschuberth — APPROVED** [review]

File: clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json (hunk 1 above)
Comment (fviernau) [reply clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json:261]: here is the follow up: https://github.com/oss-review-toolkit/ort/pull/8331

**Pull request merged**